fbchat/client.py

```diff
@@ -763,36 +764,44 @@ def fetchThreadList(self, offset=0, limit=20):
         if limit > 20 or limit < 1:
             raise FBchatUserError('`limit` should be between 1 and 20')
 
+        if type in ['inbox', 'pending', 'archived']:
+            if type == 'archived':
+                type = 'action:archived'
+        else:
+            raise ValueError('thread_type must be "inbox", "pending" or "archived"')
```

Comment:
We're using custom exceptions, so instead of `ValueError`, it's `FBchatUserError`. This was a recent change, and I'm still not sure it's a good idea, perhaps we should just drop that, and use `ValueError` instead?

Reply:
I'm not sure either :sweat_smile: 
I think I'll follow the FBchatUserError way for now. Thanks!